synapse/storage/persist_events.py

```diff
@@ -250,15 +251,17 @@ async def handle_queue_loop():
                         with PreserveLoggingContext():
                             item.deferred.callback(ret)
             finally:
-                queue = self._event_persist_queues.pop(room_id, None)
-                if queue:
-                    self._event_persist_queues[room_id] = queue
+                remaining_queue = self._event_persist_queues.pop(room_id, None)
```

Comment:
Wasn't sure of the best name here. My doesn't like it because it has a different type to the `queue` on line 233

Reply:
Probably fine, other options: `to_persist_queue`?